Source/MediaInfo/Image/File_Png.h

```diff
@@ -65,11 +65,13 @@ private :
     void IHDR();
     void PLTE() {Skip_XX(Element_Size, "Data");}
     void cICP();
-    void cLLi();
+    void cLLI();
+    void cLLi() { cLLi(); }

```

Comment:
@JeromeMartinez Saw this warning while building:
```
MediaInfoLib\Source\MediaInfo\Image\File_Png.h(69,1): warning C4717: 'MediaInfoLib::F
ile_Png::cLLi': recursive on all control paths, function will cause runtime stack overflow
```
I think you meant 
`void cLLi() { cLLI(); }` with capital `I`.

Reply:
Oops... Thanks!